bot/resources/tags/for-else.md

```diff
@@ -0,0 +1,16 @@
+**for-else**
+
+In Python it's possible to attach an `else` clause to a for loop. The code under the `else` block will be run if the `for` block is not broken out of, by either a `break`, `return`, or `raise` statement.
```

Comment:
Listing `return` or `raise` here seem odd since they are non-specific to loops, they shift the flow of execution in a different manner.

I'd rather we document it similar to [this guide](https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops) and write it something like this:

```suggestion
In Python it's possible to attach an `else` clause to a for loop. The code under the `else` block will be run when the iterable is exhausted (there are no more items to iterate over). Code within the else block will **not** run if the loop is broken out using `break`.
```